### DISCUSSION TITLE
Ways to Setup CAF to the project?

### DISCUSSION THREAD

**Discussion by PerHac13** [actor-framework/actor-framework#1971] [Q&A] [answered]
is there a way to acquire package in cmake like this
```
include(FetchContent)

# Declare and fetch CAF
FetchContent_Declare(
  caf
  GIT_REPOSITORY https://github.com/actor-framework/actor-framework.git
  GIT_TAG        master # Or replace with a specific commit/tag (e.g., "0.18.6")
)
FetchContent_MakeAvailable(caf)
```

i looking for the way to setup the cmake to project 
i want to configure the online workspace on gitpod at a moment 
looking for the best way to configure 

> Should i use docker setup 
> by configuring src code 
> or above cmake method work or not??

suggestion would appreciated

**Comment by Neverlord** [comment, marked as answer]
> is there a way to acquire package in cmake like this

Sure, our warehouse backend example is using `FetchContent`: https://github.com/interance/warehouse-backend-example/blob/main/CMakeLists.txt

**Reply by PerHac13** [reply]
are there others way to setup caf just out of curiosity??

except this and src code configure??

thanks for the help @Neverlord

**Reply by Neverlord** [reply]
CAF works like any other CMake-based project. You can include it as part of your build, install it into your system and then use `find_package`, etc. Some community members have also written recipes for package managers, but they aren't managed by us, so you need to look around to see what's available.

**Comment by PerHac13** [comment]
Also can i fork [https://github.com/interance/warehouse-backend-example](https://github.com/interance/warehouse-backend-example/blob/main/CMakeLists.txt) this repo to try out something there is no information about forking in readme @Neverlord

**Reply by Neverlord** [reply]
> Also can i fork [https://github.com/interance/warehouse-backend-example](https://github.com/interance/warehouse-backend-example/blob/main/CMakeLists.txt)

Of course.
